util/World.lua

```diff
@@ -0,0 +1,21 @@
+World               = {}
+local metaWorld     = {
+    __newindex      = function(t, k, v)
+        k = nil

```

Comment:
k, v are local variables and do not require such operations. `function() end` suits this better.

Reply:
Just wanted to make sure that no write operation is actually being done. Will simplify.